src/HeadTiltInterpreter.py

```diff
@@ -141,11 +149,21 @@ def run(self):
 
         # Read data and decide movement
         while True:
-            x_ra, z_ra = self.get_cluster_avg(sock)  # Rolling averages for live clusters
+            x_ra, z_ra = self.get_cluster_avg(accel_socket)  # Rolling averages for live clusters
             dominant = self.calc_dominant_axis(x_ra, z_ra, axis_ranges)
             x_percentage, z_percentage = self.get_axis_percentage(
                 dominant, x_ra, z_ra, axis_ranges)
-            print('X Percentage: {} | Z Percentage: {}'.format(x_percentage, z_percentage))
+
+            if config.OUTPUT_DATA:
+                dict = {
+                    'tilt': z_percentage,
+                    'roll': x_percentage
+                }
+                send_json = json.dumps(dict)
+                output_socket.send_string(send_json)
```

Comment:
@BenjaminHunt I think we might as well use the socket.send_json() method here to reduce verbosity on our end.

Reply:
That makes sense. I didn't realize that was an option. I'll open a refactoring branch to make some improvements in the code and I'll take care of this within it.

Reply:
I pushed the change to master @AdamVD